src/frontend/magnify/src/components/design-system/Card/MagnifyCard.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import MagnifyCard from './MagnifyCard';
+
+describe('MagnifyCard', () => {
+  it('should render successfully a MagnifyCard', async () => {
+    render(
+      <MagnifyCard title={'Test'}>
+        <div>Test body</div>
+      </MagnifyCard>,
+    );
+    const header = screen.getByRole('heading', { level: 3 });
+    expect(header).toBeInTheDocument();
+    expect(header).toHaveTextContent('Test');
```

Comment:
Should do the job in one line.
```suggestion
    screen.getByRole('heading', { level: 3, name: 'Test' });
```